### ISSUE TITLE
CMD command: make a new conversation as a real shell

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#67]
```
Create a new command like shell that will open a new MUC.

All messages sent goes to stdin and all results - sterr or stdout - are 
response in the MUC.

Create a new shell syntax to emulate Ctrl+C to break the current command


```

Original issue reported on code.google.com by `Florent....@gmail.com` on 29 Jan 2011 at 7:07

**Comment by GoogleCodeExporter** [comment]
```
GTalkSMS now replies back to the full jid or MUC the request came from. This 
should it make pretty easy to implement this feature.
```

Original comment by `fschm...@gmail.com` on 11 Mar 2011 at 2:43

**Comment by GoogleCodeExporter** [comment]
```
This feature could be so cool !
Because I sometimes need to access my terminal trough 3G, It could be nice to 
implement a new command : shell
then every command would be interpreted like cmd:command !
to exit just need to send exit ;)
The only "borring things" is because up/Down Arrow or Tab could not be passed 
throught gtalk .. 
```

Original comment by `aurelien...@gmail.com` on 25 May 2011 at 11:48

**Comment by GoogleCodeExporter** [comment]
```
We can now open terminals in different MUC
```

Original comment by `Florent....@gmail.com` on 11 Jul 2011 at 9:26
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v3.0
```

Original comment by `Florent....@gmail.com` on 15 Jul 2011 at 3:35
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
